utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -174,17 +175,18 @@ def run_indv(self, suffix="base", coupler_log_path=None, st_archive=False):
         logger.info(infostr)
 
         case_run(self._case)
-        if st_archive:
```

Comment:
Important change: Do archiving as last step in this function. Removes the need for coupler_log_path.